src/styles/index.css

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card .name {
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-top: 40px;
+  margin-bottom: 0;
+}
+
+.card .code {
```

Comment:
The 'code' class should follow BEM naming conventions. It should be 'card__code'.